client/my-sites/plans/jetpack-plans/product-store/hooks/use-store-item-info.tsx

```diff
@@ -130,9 +133,19 @@ export const useStoreItemInfo = ( {
 
 	const getIsIncludedInPlan = useCallback(
 		( item: SelectorProduct ) => {
+			// If user owns the Jetpack Complete Plan/bundle, then JetPack Security plan/bundle should
+			// be considered as included in the Complete plan ("Part of the current plan").
+			if (
+				( JETPACK_COMPLETE_PLANS as ReadonlyArray< string > ).includes(
+					( sitePlan && sitePlan.product_slug ) || ''
+				) &&
+				( JETPACK_SECURITY_PLANS as ReadonlyArray< string > ).includes( item.productSlug )
+			) {
+				return true;
+			}
```

Comment:
[Non-blocking]

This conditional return can be rewritten to use two shorter (and perhaps more readable?) variable expressions:

```suggestion
			const siteHasCompletePlan =
				sitePlan?.product_slug &&
				( JETPACK_COMPLETE_PLANS as ReadonlyArray< string > ).includes(
					sitePlan.product_slug
				);
			const itemIsSecurity =
				( JETPACK_SECURITY_PLANS as ReadonlyArray< string > ).includes(
					item.productSlug
				);

			if ( siteHasCompletePlan && itemIsSecurity ) {
				return true;
			}
```

Reply:
Thanks @mattgawarecki , yes thats more readable, I like it, thanks!!  LGTM! 👍